src/Place/Canonical/ImportDuplicatePlacesProcessor.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Place\Canonical;
+
+class ImportDuplicatePlacesProcessor
+{
+    private DuplicatePlaceRepository $duplicatePlaceRepository;
+    private DuplicatePlaceRemovedFromClusterRepository $duplicatePlacesRemovedFromClusterRepository;
+
+    public function __construct(
+        DuplicatePlaceRepository $duplicatePlaceRepository,
+        DuplicatePlaceRemovedFromClusterRepository $duplicatePlacesRemovedFromClusterRepository
+    ) {
+        $this->duplicatePlaceRepository = $duplicatePlaceRepository;
+        $this->duplicatePlacesRemovedFromClusterRepository = $duplicatePlacesRemovedFromClusterRepository;
+    }
+
+    public function sync(): void
+    {
+        $this->deleteOldClusters();
```

Comment:
I guess this is weird, but will get more method calls in the next pull requests?

Reply:
Yes :-)